src/ocaml/preprocess/parser_raw.mly

```diff
@@ -4744,7 +4672,7 @@ atat_mode_expr:
 
 at_modalities_expr:
   | AT modalities {$2}
-  | AT error { expecting $loc($2) "modality expression" }
+  // | AT error { expecting $loc($2) "modality expression" }
```

Comment:
Why comment this out?

Reply:
Merlin's error recovery when parsing works differently than in the compiler. You'll notice there's other similar things commented out.